Short-Answer/Algorithms_Answers.md

```diff
@@ -3,12 +3,13 @@
 ## Exercise I
 
 a) O(n) -- linear (grows in direct proportion of the input size)
-
+O(1) * O(n^3) * O(n^2)
 
```

Comment:
in this case remember, we are trying to calculate the rate of calculations, so top is n^3, however in the loop you are multiplying the constant with n^2, think of 2^3, and reassigning a to 0+2^2, which will be 4,  if you can try to setup some pseudo examples, you would want to divide in this case, when you divide exponents, you subtract.